x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -399,6 +399,13 @@ export type HostMetadata = Immutable<{
   '@timestamp': number;
   event: {
     created: number;
+    kind: string;
+    id: string;
+    category: string;
```

Comment:
`category` and `type` can be an array per the ecs spec. I believe the endpoint will actually send these values as an array of a single value for metadata.